### ISSUE TITLE
URI prefill only search by origin

### ISSUE THREAD

**Issue by sandysage** [mozilla-lockbox/lockbox-extension#627]
When I land on my Gmail account, I want to only see the relevant results for https://mail.google.com/, and not all my google accounts and properties.

**Comment by linuxwolf** [comment]
@sandysage 

I'm not sure this makes sense, and contradicts the notes/gist references in #527.  This is particularly true for sites that have separate sub-domains for "desktop" (almost always "www."; e.g., "www.facebook.com") versus "mobile" (e.g., "mobile.facebook.com").

With the above, the results may be "no entries" when the user's perception of their entries is that there is (at least) one.

**Comment by linuxwolf** [comment]
I think I now see the intent, which is to filter the doorhanger by only site name (origin) when it is first opened.

**Comment by jimporter** [comment]
@linuxwolf Yeah, I should probably go in and update the original comment with some more details. I think I alluded to it elsewhere, but the idea is that we *shouldn't* match on, say, the username field.
